js/subscribe.js

```diff
@@ -0,0 +1,44 @@
+const modal = document.getElementById("submitModal");
+const submitSubscription = document.getElementById("submitSubscription");
+const congratsText = document.getElementById("congrats");
+const emailInput = document.getElementById("emailAddress");
+
+// function subscriptionButton() {
+//     modal.style.display = "block";
+// }
+
+const modalOptYes = document.getElementById("modalYes");
+const modalOptNo = document.getElementById("modalNo");
+
+
+
+const validateEmail = () => {
```

Comment:
This function is also available in homepage.js. Please D.R.Y. Try to have a common file where both the homepage and subscribe page can access this function